com.librumreader.librum.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.librumreader.librum
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: librum
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission for an ebook reader.

Reply:
From the docs I got that "--share=ipc - share IPC namespace with the host (necessary for X11)". Since my application is supposed to run on X11, I supposed that I need to have it in my permissions. I need `--share=network` since my application is syncing books and other kind of data to the server, thus needs internet connection for http requests.
`--filesystem=host` is needed since my application will load selected files from the user's file system and export the files from the application if wanted by the user. From what I understood the `--socket=...` is for showing windows? Since my application is a GUI application I supposed that I need that permission as well.

Please correct me if I misunderstood any permission. Also would my application run on both X11 and wayland the way my permissions are set up? I did not get if `--socket=fallback-x11` tries wayland and then falls back to x11 and thus covers both wayland and x11, or if I need also `--socket=x11` and `--socket=wayland`.

Reply:
> very broad filesystem permission

there is only ONE line that is a filesystem permission here. Its line 10.

Reply:
> > very broad filesystem permission
> 
> there is only ONE line that is a filesystem permission here. Its line 10.

My bad, I didn't read the "filesystem". I chose `host` since my application is not supposed to care where the user wants their file installed to or imported from, it can be from a home directory or from a privileged location.

Reply:
this is too broad. it's an ebook reader. Since it's Qt based, in theory it should use the portal by default, and not need filesystem performission. Otherwise `host` is very broad.